src/Main.java

```diff
@@ -32,28 +32,26 @@ public static int getSumSalaryMonth() {
         }
         return summa;
     }
-    public static int findEmploeerWithMinSalary() {
+    public static String findEmploeerWithMinSalary() {
         int minimum = employees[0].getSalary();
         int emp = 0;
         for (int i = 1; i < employees.length; i++) {
             if (minimum > employees[i].getSalary()) {
-                minimum = employees[i].getSalary();
                 emp = i;
             }
         }
-        return emp;
+        return employees[emp].getFio();
```

Comment:
надо возвращать сотрудника
```suggestion
        return employees[emp];
```